dashboard/templates/dashboard/index.html

```diff
@@ -56,7 +56,7 @@
                 </nav>
             {% elif user_type == constant_ut_adminStaff %}
                 <nav class="nav d-flex">
-                    <a class="p-2 text-muted active" href="#student_applications" data-toggle="tab">Student
+                    <a class="p-2 text-muted active" href="{% url 'dashboard:admissions:index'%}" data-toggle="tab">Student
```

Comment:
You should be passing the "user_type" as 'ut_admin_staff' for now from this URL (the way HighSchool all school navigation is taken care of from dashboard) which will be changed into a user object (once we figure that part out) in the coming sprint (high priority for next sprint).

Reply:
I don't see why that is necessary. Ideally speaking once we have the sessions working, we should take the user_type from there and not pass it around. So it doesn't make much sense to pass it in your case as well.